kloppy/tests/test_statsbomb.py

```diff
@@ -254,6 +254,36 @@ def test_foul_committed(self, lineup_data: Path, event_data: Path):
 
         assert len(dataset.events) == 23
 
+    def test_own_goal(self, lineup_data: Path, event_data: Path):
```

Comment:
I think it would be good if we also added a _score state_ test for a scenario where an own goal is scored. I presume [the current test](https://github.com/PySport/kloppy/blob/master/kloppy/tests/test_state_builder.py#L18) only tests the happy flow where only regular goals are scored.